tests/contrib/anthropic/test_anthropic_llmobs.py

```diff
@@ -15,6 +15,14 @@
 the necessary information to make the API call.\n\nNo other tools are needed to answer this question. \
 We can proceed with calling the get_weather tool with the provided location.\n</thinking>'
 WEATHER_OUTPUT_MESSAGE_2 = '[tool: get_weather]\n\n{"location": "San Francisco, CA"}'
+WEATHER_OUTPUT_MESSAGE_2_Modified = [
```

Comment:
```suggestion
WEATHER_OUTPUT_MESSAGE_2 = [
```
Do we still need the old weather output message 2?

Reply:
just testing for now - will delete later